### PR TITLE
Just test for form added in first-app2 fix?

### PR DESCRIPTION
**Pull request by Mypeterhub**

### CODE REVIEW

**Comment by Mypeterhub** [comment]
New commit for first-app3 working!!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    
+     
+    <div id="button">
+    <p>
+    <button id="ajL" type="button">Ajax Load</button>
+    <button id="loL" type="button">Local Load</button>
+    <button id="loS" type="button">Local Save</button>
+    <button id="loC" type="button">Local Clear</button>
+    </p></div>
+    <div id="wonders">
```
Comment (mbMosman) [inline index.html]: Missing aria attributes for accessibility.

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            } else if (nameValidate.length < 2) {
+
+                alert("Need a valid name at least 2 charactors!");
+
+            } else if (!validateEmail(emailValidate)) {
+
+                alert("Email is invalid!");
+            }
+
+            function validateEmail(emailValidate) {
```
Comment (mbMosman) [inline js/script.js]: Nice work with the additional form feature.

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                } else {
+                    return false;
+                }
+            }
+
+        });
+
+    });
+
+    $("#ajL").click(touristSite.touristSitePlace);
```
Comment (mbMosman) [inline js/script.js]: These could also be inside of the document.ready function above.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+            var object = data.touristSite;
+            console.log(object);
+
+            var listLocation = $("#wonders");
+            var touristCount = object.length;
+            var target = $("#wonders");
+            target.innerHTML = ""
+
+            if (touristCount > 0) {
+                for (i = 0; i < touristCount; i = i + 1) {
```
Comment (mbMosman) [inline js/script.js]: The jQuery each could be used instead of the loop.

**Comment by mbMosman** [comment]
Overall nice work, but submitted late.

**Pull request closed without merging**